scripts/criu-ns

```diff
@@ -80,8 +80,20 @@ def run_criu(args):
     Spawn CRIU binary
     """
     print(sys.argv)
-    os.execlp('criu', *['criu'] + args)
-    raise OSError(errno.ENOENT, "No such command")
+
+    if "--criu-binary" in args:
+        try:
+            path = args[args.index("--criu-binary") + 1]
+            args.remove("--criu-binary")
+            args.remove(path)
+            args.insert(0, "criu")
+            os.execv(path, args)
+            raise OSError(errno.ENOENT, "No such command")
+        except (ValueError, IndexError):
+            raise OSError(errno.ENOENT, "--criu-binary missing argument")
+    else:
+        os.execlp("criu", *["criu"] + args)
+        raise OSError(errno.ENOENT, "No such command")
```

Comment:
Changes to criu-ns should go to separate commit and changes to test should go to another separate commit as logically separate changes. Please separate them and force push.

Reply:
I just pushed the `criu-ns` related code into a separate commit. Thanks!